src/modules/common/components/uploadJSONInput/uploadJSONInput.js

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
```

Comment:
NRTTL
Getting this error in the console when performing the flow
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/57dabdef-7a09-40ed-8e3f-6a10bddfac1b)

Reply:
Fixed